mkdocs.yml

```diff
@@ -340,7 +340,7 @@ nav:
           - 谓词函数: 3.ngql-guide/6.functions-and-expressions/8.predicate.md
```

Comment:
![image](https://user-images.githubusercontent.com/78209557/198541668-3f67ad71-bc39-4662-a52d-bf0b0b738b13.png)
If we need to change the macro in the above pic?